src/plugins/stepsform/stepsform.js

```diff
@@ -0,0 +1,216 @@
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var eventTarget = wb.init( event, componentName, selector ),
+			elmId;
+
+		if ( eventTarget ) {
+			elmId = eventTarget.id;
+
+			// Ensure there is a unique id on the element
+			if ( !elmId ) {
+				elmId = componentName + "-id-" + idCount;
+				idCount += 1;
+				eventTarget.id = elmId;
+			}
+
+			// Only initialize the i18nText once
+			if ( !i18nText ) {
+				i18n = wb.i18n;
+				i18nText = {
+					prv: i18n( "prv" ),
+					nxt: i18n( "nxt" )
+				};
+			}
+
+			// Only initialize the buttons once
+			if ( !btnPrevious || !btnNext || !btnSubmit ) {
+				btnPrevious = createStepsButton( "button", "button", "mrgn-rght-sm mrgn-bttm-md", i18nText.prv ),
+				btnNext = createStepsButton( "button", "submit", "mrgn-bttm-md", i18nText.nxt ),
+				btnSubmit = createStepsButton( "input", "submit", "mrgn-bttm-md" );
+			}
+
+			/*
+			 * Variable and function definitions
+			 * These well be initialized once per instance of plugin.
+			 */
+			var form = eventTarget.getElementsByTagName( "FORM" )[ 0 ],
+				fieldsets = ( form ) ? $( form ).children( "fieldset" ) : 0,
+				settings = wb.getData( eventTarget, componentName ),
+				hasStepsInitialized;
+
+			/*
+			 * Determines if html is correctly formatted and initialize all fieldsets/legend combitions into steps.
+			 */
+			for ( var i = 0, len = fieldsets.length; i < len; i++ ) {
+
+				/*
+				 * Variable and function definitions
+				 * These well be initialized once per instance of each fieldset.
+				 * Determines the following business rules:
+				 *  -Only allow steps if elements are in proper order fieldset -> legend -> div
+				 *  -Only allow NEXT button on first step
+				 *  -Only allow final SUBMIT button on last step
+				 */
+				var fieldset = fieldsets[ i ],
+					isFirstFieldset = ( i === 0 ) ? true : false,
+					isLastFieldset = ( i === ( len - 1 ) ) ? true : false,
+					legend = fieldset.firstElementChild,
+					div = ( legend && legend.tagName.toUpperCase() === "LEGEND" ) ? legend.nextElementSibling : false;
+
+				if ( div && div.tagName.toUpperCase() === "DIV" ) {
+					var btnClone;
+					hasStepsInitialized = true;
+
+					if ( !isFirstFieldset ) {
+						btnClone = btnPrevious.cloneNode( true );
+						btnClone.id = componentName + "-btn-" + idCount;
+						idCount += 1;
+
+						setStepsBtnEvent( btnClone );
+						div.appendChild( btnClone );
+					}
+
+					if ( !isLastFieldset ) {
+						btnClone = btnNext.cloneNode( true );
+						btnClone.id = componentName + "-btn-" + idCount;
+						idCount += 1;
+
+						setStepsBtnEvent( btnClone );
+						div.appendChild( btnClone );
+					} else {
+						var btnValue = ( settings && settings.submitBtnVal ) ? settings.submitBtnVal : "X",
```

Comment:
- [ ] i18n Text (add "Submit" as default button text (enabling tmpl-submit (not sure about this)))

Could also use `<i class="glyphicon glyphicon-send" aria-hidden="true"></i>` but accessibility would be an issue.